students/Deniss_Semcovs/Lesson04/mailroom_part2.py

```diff
@@ -26,52 +52,109 @@ def sendTYnote():
                 if donName == i[0]:
                     donation = int(input("Please enter the donation amount: "))
                     i.append(donation)            
-                    print("Hello {}, thank you for your generous donation of ${} to support our cause.".format(donName, donation))
                     break
             else:
 				newDonation = int(input("This is a new donor, please enter the donation amount: "))
 				data.append([donName, newDonation])
 				print("Letter for the donor: ")
                 print("Hello {}, thank you for your generous donation of ${} to support our cause.".format(donName, newDonation))
-            break
-
-def createReport():
+            break    
+	
+def create_report():
```

Comment:
Missing the sorting feature - donors should be listed by total given, in descending order